doc/source/cluster/kubernetes/getting-started/raycluster-quick-start.ipynb

```diff
@@ -0,0 +1,595 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "2248a2d8-bd70-4ac3-8fa8-7beeb4a2a8ae",
+   "metadata": {
+    "collapsed": true,
```

Comment:
what does this mean?

Reply:
Actually, I don't know. Maybe some hidden cells or metadata added by Jupyter Notebook? But it's not important since it doesn't show up in the doc or affect the tests.